core/src/test/java/com/linecorp/armeria/common/DefaultRequestIdTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+class DefaultRequestIdTest {
+
+    @Test
+    void basic() {
+        final RequestId id = RequestId.random();
+        assertThat(id).isInstanceOf(DefaultRequestId.class);
+        assertThat(id.longText()).hasSize(16);
```

Comment:
Think it's worth having a test that passes in a value and checks the actual values of `longText`, `shortText`, especially with a positive number

Reply:
Agreed. Added a test case for that.